packages/flutter/test/material/stepper_test.dart

```diff
@@ -716,4 +716,66 @@ void main() {
 
     expect(tester.takeException(), isNull);
   });
+
+  testWidgets('Stepper custom elevation', (WidgetTester tester) async {
+    const double elevation = 4.0;
+
+    await tester.pumpWidget(
+      MaterialApp(
+        home: Material(
+          child: Container(
+            width: 200,
+            height: 75,
+            child: Stepper(
+              type: StepperType.horizontal,
+              horizontalElevation: elevation,
+              steps: const <Step>[
+                Step(
+                  title: Text('Regular title'),
+                  subtitle: Text('Text subtitle'),
+                  content: Text('Text content'),
+                ),
+              ],
+            ),
+          ),
+        ),
+      ),
+    );
+
+    await tester.pump();
+    final Stepper stepper = find.byType(Stepper).evaluate().first.widget as Stepper;
```

Comment:
You can also get the `Stepper` like

```dart
final Stepper stepper = find.widget(find.byTpe(Stepper));
```

Since your widget tree will only have a single instance of `Stepper` you don't need to worry exceptions.

Reply:
Done